src/Model/Comment.php

```diff
@@ -865,6 +865,14 @@ public function ReplyForm()
         return $controller->ReplyForm($this);
     }
 
+    /**
+     * @return DBDatetime
+     */
+    public function getDate()
+    {
+        return $this->dbObject('Created');
```

Comment:
How comes we are returning the `dbObject`? Usually these `get*` methods would return a string and have casting for the template.

Reply:
Yep, fair enough!